test/test_sparse_csr.py

```diff
@@ -3364,9 +3310,123 @@ def test_sparse_to_sparse_compressed(self, device, dtype, coalesced, layout):
             self.assertEqual(torch.tensor(sp_matrix.data), pt_matrix.values())
 
 
+class _TritonLibrary(object):
+    lib = torch.library.Library("triton", "DEF")
+    ops = {}
+
```

Comment:
So, basically, library recreation in tests run sequentially in a batch caused the issue described above...